source/local/local.go

```diff
@@ -108,19 +151,61 @@ func runHook(file string) (map[string]string, error) {
 		if err != nil {
 			return features, err
 		}
+		features = bytes.Split(stdout.Bytes(), []byte("\n"))
```

Comment:
`lines` here as well – I wouldn't call it features, yet, as the data hasn't really been parsed

Reply:
Done